data/human/human missions.txt

```diff
@@ -7455,3 +7455,573 @@ mission "Adelita 5"
+			`	"<first> <last>! My bank contact tells me you've been doing quite well for yourself, and I've come across an even bigger investment opportunity than last time! Meet me at my new office on <planet> (note the address, I've moved up since my promotion). Looking forward to chat!"`
+			`		Howard Vendeur`
+			`		Investor Relations Manager, Megaparsec Incorporated`
+			choice
+				`	(Message him back that you're on the way.)`
+				`	(Send a message congratulating him on the promotion.)`
+				`	(Ignore and delete the message.)`
+					decline
+			`	You immediately receive an automated response:`
+			`	"Dear Valued Contact,`
+			`	Thank you for reaching out. I am currently out of the office and will respond to your message as soon as possible.`
+			`	For urgent matters, please contact our support team."`
+			`		Howard Vendeur`
+			`		Investor Relations Manager, Megaparsec Incorporated`
+				accept
+	on complete
+		conversation
+			`You land <ship> on <planet> and make your way to Howard's office at the Megaparsec Incorporated satellite office near the Syndicate's headquarters.`
+			`	You find yourself waiting in a sleek, sterile lobby, with the hum of distant machinery filling the air. The walls are lined with glossy advertisements for the various fast and lightweight ships, aircraft, and hovercraft that Megaparsec is known for, while a polite receptionist assures you that Howard will be with you "shortly," though you suspect that could mean anything from minutes to hours.`
+			choice
+				`	(Take a seat in the lobby.)`
+			branch makeemwait
+				has "syndicate business: mean to Howard"
+			`	Much to your surprise, the wait is brief. Within minutes, Howard emerges with a warm smile, extending a hand as he welcomes you into his office, clearly eager to discuss the next big deal.`
+				goto next
+			label makeemwait
+			`	Hours crawl by as the polished receptionist offers increasingly strained reassurances. Finally, Howard arrives, barely masking his annoyance, and motions for you to follow with a curt nod, clearly viewing this meeting as an inconvenience.`
+			label next
+			`	The inside of Howard's office is modest yet stylish, with a sleek table, a few ergonomic chairs, and a large screen cycling through market trends. The minimalist decor is designed to impress, but it's clear this is more for show than substance.`
+			`	Howard gestures for you to take a seat as he settles behind his desk. His demeanor shifts slightly, and he leans forward, lowering his voice as if sharing a secret.`
+			`	"You know I've called you here for an incredible investment opportunity," Howard begins. "But before we get into that, I need a small favor. It's nothing too complicated, but there's too many prying eyes here. I'm wrapping up some meetings today, but meet me at the spaceport tomorrow. I'll fly in my shuttle as your escort to Sunracer. I have a few work calls to take on the way, but I'll explain everything once we get there."`
```

Comment:
You can't make time pass with missions just yet. Do you have an event to meet him?

Reply:
Ah, I just assumed every time you go to the spaceport a day passes. Would it be better to change `tomorrow` to just `later` and sidestep that whole issue?

Reply:
I think so, yes.